specification/monitor/resource-manager/Microsoft.Insights/stable/2018-03-01/metricAlert_API.json

```diff
@@ -319,7 +319,13 @@
           },
           "204": {
             "description": "No content: the request was successful, but the response is empty"
-          }
+          },
+          "default": {
+            "description": "BadRequest",
+            "schema": {
+              "$ref": "../../stable/2016-03-01/alertRules_API.json#/definitions/ErrorResponse"
```

Comment:
It seems not a good practice to refer to another version (but as it is already done, there might not possible to fix within this version).

Also the response schema is not correct as in LintDiff. It should have an error property, which then contains code and message.
Though it is likely not possible to fix within same version, you might need to suppress.

Reply:
@weidongxu-microsoft right.
I've requested a suppression for the LintDiff and also mentioned it in the swagger.
What about the sdk check which fails, what should i do with this? suppression as well?

Reply:
You can ignore the CI that without "required" label. They does not block the merge.

As dotnet build, it appears from its log above, fails on some class as `azure-sdk-for-net/sdk/monitor/Microsoft.Azure.Management.Monitor/tests/BasicTests/DataCollectionRulesTests.cs`, so it is likely not caused by your PR.

Reply:
@weidongxu-microsoft  cool, so is there anything that blocking me to merge this PR?

Another thing, I've another PR (https://github.com/Azure/azure-rest-api-specs/pull/12234) with some error about Swagger ApiDocPreview failed, should i fixed this or it's like the sdk?

Reply:
I tried to re-run ApiDocPreview CI in the other PR.

Both PR still have LintDiff, which blocks. I think perhaps the suppress is not done correctly. The correct one is perhaps
```
  - suppress: R4007
```

Reply:
I think you're right, fixed this and now the LintDiff passed

Reply:
@weidongxu-microsoft  btw, i've few more PRs which waiting a long time, can you please take a look also?
https://github.com/Azure/azure-rest-api-specs/pull/12232
https://github.com/Azure/azure-rest-api-specs/pull/12233
https://github.com/Azure/azure-rest-api-specs/pull/9790

Reply:
For https://github.com/Azure/azure-rest-api-specs/pull/9790 you might again ping the owner.

For the rest on me I will merge them. However after first PR merge there is conflict on the other PRs. You need to resolve the conflict as I do not have permission to commit to your forked repo.

Please resolve 1 PR, ping me, I will merge it. Then proceed on next. They all changes the same readme.md and hence got conflict against each other.

Reply:
cool, saw you merged one. resolved conflict here: https://github.com/Azure/azure-rest-api-specs/pull/12234
can you please merge?

Reply:
done 12234

Reply:
thanks, resolved https://github.com/Azure/azure-rest-api-specs/pull/12233
merge?

Reply:
12233 still has the conflict. You probably need to refresh master every time before merge it to branch.

Reply:
or just click commit after resolving :)
make sure i resolved it correctly and you can merge it.

Reply:
12233 merged

Reply:
great, thanks!
so this is the only one left (and also the old one which you're not own)
i have to wait a bit with this one, cause maybe we need to fix a small thing, so i guess it will be tomorrow or next week.
i'll ping you here :)

Reply:
This PR 12231 could also be resolved and merged.

Reply:
yea, i'm talking about this one which i have to be a small change in it before merging

Reply:
@weidongxu-microsoft resolved conflict in current PR and we can merge it as well :)
btw, what can i do with the other PR that i've that the reviewer doesn't answer after 2 pings?
https://github.com/Azure/azure-rest-api-specs/pull/9790

Reply:
"Swagger ModelValidation" failed on this. I could not get the cause. I will check around.

I've merge the other one.

Reply:
@weidongxu-microsoft  Thanks a lot for all that work!
Can you please take a look at this one? https://github.com/Azure/azure-rest-api-specs/pull/12314
I can't figure out what's the issue with the LintDiff

Reply:
Not sure. My guess is that if you have multiple json, and they all defines `ApiVersionParameter`, you need to make it identical includes its `description`.

Reply:
@weidongxu-microsoft thanks, i'm not sure that was a real problem, but solve this and now there is another issue that i'm not sure how to deal with, can you please take another look? 
https://github.com/Azure/azure-rest-api-specs/pull/12314

Reply:
[microsoft.insight.txt](https://github.com/Azure/azure-rest-api-specs/files/5802257/microsoft.insight.txt)
attached our operations API which contains this API, not sure why this validation fails

Reply:
I think it asks for `operations` endpoint like this https://github.com/Azure/azure-rest-api-specs/blob/master/specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/operation.json#L37?

Reply:
@weidongxu-microsoft We haven't operations_API.json file in any of our working APIs, for example this one is the same API with different version and without operations_API.json file and we've completed a PR on this one last week:
https://github.com/Azure/azure-rest-api-specs/tree/master/specification/monitor/resource-manager/Microsoft.Insights/stable/2019-03-01
do you have any idea who can i ask such questions? i'm blocked on this issue :/

Reply:
Here is the requirement https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/openapi-authoring-automated-guidelines.md#r3023

Some validation only apply to new api-version.

You can check with "akning-ms" on this.

Reply:
what is akning-ms and how can i contact them?

Reply:
This is his github ID. Though I do not think it will be exempted. And monitor had the json https://github.com/Azure/azure-rest-api-specs/blob/master/specification/monitor/resource-manager/Microsoft.Insights/stable/2015-04-01/operations_API.json

Reply:
@weidongxu-microsoft 
cool, i'll try him.

I've 2 additional PRs that i want to merge, but experiencing the same model validations we had in the current PR,
can you do the same process you did for this one to merge them?
https://github.com/Azure/azure-rest-api-specs/pull/12371
https://github.com/Azure/azure-rest-api-specs/pull/12372

Reply:
For these you can also contact Arthur.

Reply:
sure, will do.
but first you don't have to verify that the failing model validation are false alerts?

Reply:
@weidongxu-microsoft talked with Ray and suppressed the modelValidation, can you please help me merge this one?
https://github.com/Azure/azure-rest-api-specs/pull/12371